ui/mainwindow.ui

```diff
@@ -89,7 +91,7 @@ background-position:bottom right;
      <x>0</x>
      <y>0</y>
      <width>829</width>
-     <height>27</height>
+     <height>39</height>
```

Comment:
Designer really likes to mess with dimensions when you edit something unrelated. Personally, I usually discard these using git after the fact. But I'm not forcing you to do it, just wanted to point it out - you already have my :+1:  :)